mass_mailing_switzerland/models/mass_mailing.py

```diff
@@ -29,7 +29,7 @@ class MassMailing(models.Model):
     partner_test_sendgrid_id = fields.Many2one('res.partner',
                                                'Test Partner')
 
-    _sql_constraints = [('slug_uniq', 'unique (mailing_slug)',
+    _sql_constraints = [('slug_uniq', 'unique (slug)',
                          'You have to choose a new slug for each mailing !')]
```

Comment:
In fact I checked and this field was removed! So you can remove the constraint as you did before, sorry.